lib/repo-type-mappings/index.js

```diff
@@ -19,18 +21,25 @@ const getDirectories = (source) => {
 const mappings = getDirectories(__dirname)
 
 const exportedMappings = _.reduce(mappings, (exportedMappings, dirName) => {
-  const configFile = readFileSync(join(dirName, 'versionist.conf.js'), 'utf8')
   const dependencies = YAML.safeLoad(readFileSync(join(dirName, 'dependencies.yml'), 'utf8'))
-
   const type = basename(dirName)
   exportedMappings[type] = {}
   exportedMappings[type].getConfig = (options) => {
+
+    let configuration = '';
+    try {
+      configuration = readFileSync(CONF_FILENAME, 'utf8');
```

Comment:
this reads the file relative to the CWD. Shouldn't we ensure it's relative to package.json?

Reply:
Hm... Good point. It's expecting it on the root. `package.json` is not necessarily in the root though. I used the path as passed in from CLI and tested it.